conans/test/unittests/util/tools_test.py

```diff
@@ -729,6 +732,87 @@ def get_file():
 
         thread.stop()
 
+    @attr('slow')
+    @attr('local_bottle')
+    def test_get_mirror(self):
+        """ tools.get must supports a list of URLs. However, only one must be downloaded.
+        """
+
+        import gzip
+        # Create a tar file to be downloaded from server
+        tmp = temp_folder()
+        files = ["test.txt.gz", "foo.txt.gz", "bar.txt.gz"]
+        checksums = {"md5":[], "sha1": [], "sha256": []}
+        for file in files:
+            filepath = os.path.join(tmp, file)
+            with gzip.open(filepath, "wb") as f:
+                f.write(file.encode())
+            checksums["md5"].append(md5sum(filepath))
+            checksums["sha1"].append(sha1sum(filepath))
+            checksums["sha256"].append(sha256sum(filepath))
+
+        thread = StoppableThreadBottle()
```

Comment:
I think that we are already checking that `get()` and `download()` are able to work and the downloaders/requesters return the files from the server. IMO it is enough to check the added functionality: we should mock the `downloader.download` function and check the number of times it is called and the arguments passed to it, no need to mock a server, this can be an actual unittest.

Reply:
> we should mock the downloader.download function and check the number of times it is called and the arguments passed to it, no need to mock a server, this can be an actual unittest.

That's an excellent idea